bundles/org.openhab.binding.dmx/src/main/java/org/openhab/binding/dmx/internal/handler/ColorThingHandler.java

```diff
@@ -131,29 +131,29 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                         }
                         targetValueSet = turnOffValue;
                     }
-                } else if (command instanceof HSBType) {
+                } else if (command instanceof HSBType hsb) {
```

Comment:
Consider for full consistency:
```suggestion
                } else if (command instanceof HSBType hsbCommand) {
```